src/main/resources/messages_en_GB.properties

```diff
@@ -0,0 +1,14 @@
+code=en_GB
+error.404.message=Oops! The page you are looking for does not exist.
+error.404.title=404 - Page Not Found
+error.bad_login_method.message=Your authentication does not permit you access to this resource
+error.bad_login_method.title=Unauthorized
+index.count=You are visitor number {0}.
+index.hello=Hello {0}!
+
+bookings.description=Description
+bookings.start=Start
+bookings.end=End
+bookings.share=Share Room?
+bookings.priority=Priority
+booking.ifempty=No bookings available
```

Comment:
Why do we need a dedicated briish translation?
Since we configured the server locale to be english, the default should be the english translation of messages.properties

Reply:
I am on a briish system and my im too lazy to reconfigure it, im sure some of our future users may relate xd

Reply:
I just noticed the file name is wrong too, so this didn't even get used.
I'll just remove it alongside my other style nicks.